### PR TITLE
Settlement subresources php examples

### PR DESCRIPTION
**Pull request by sandervanhooft**

### CODE REVIEW

File: source/reference/v2/settlements-api/list-settlement-captures.rst [hunk 1]
```diff
@@ -98,6 +98,16 @@ Example
...
 
+   .. code-block:: php
+      :linenos:
+
+      <?php
+      $mollie = new \Mollie\Api\MollieApiClient();
+      $mollie->setAccessToken("access_Wwvu7egPcJLLJ9Kb7J632x8wJ2zMeJ");
+      
+      $settlement = $mollie->settlements->get("stl_jDk30akdN");
+      $payments = $settlement->captures();
```
Comment (ndijkstra) [inline source/reference/v2/settlements-api/list-settlement-captures.rst]: Variable should be `$captures`

File: source/reference/v2/settlements-api/list-settlement-chargebacks.rst [hunk 2]
```diff
@@ -40,6 +40,16 @@ Request
...
 
+   .. code-block:: php
+      :linenos:
+
+      <?php
+      $mollie = new \Mollie\Api\MollieApiClient();
+      $mollie->setAccessToken("access_Wwvu7egPcJLLJ9Kb7J632x8wJ2zMeJ");
+      
+      $settlement = $mollie->settlements->get("stl_jDk30akdN");
+      $payments = $settlement->chargebacks();
```
Comment (ndijkstra) [inline source/reference/v2/settlements-api/list-settlement-chargebacks.rst]: Variable should be `$chargebacks`

File: source/reference/v2/settlements-api/list-settlement-refunds.rst [hunk 3]
```diff
@@ -40,6 +40,16 @@ Request
...
 
+   .. code-block:: php
+      :linenos:
+
+      <?php
+      $mollie = new \Mollie\Api\MollieApiClient();
+      $mollie->setAccessToken("access_Wwvu7egPcJLLJ9Kb7J632x8wJ2zMeJ");
+      
+      $settlement = $mollie->settlements->get("stl_jDk30akdN");
+      $payments = $settlement->refunds();
```
Comment (ndijkstra) [inline source/reference/v2/settlements-api/list-settlement-refunds.rst]: Variable should be `$refunds`

**Comment by sandervanhooft** [comment]
Thanks @ndijkstra ! Fixed it.

**Pull request merged**